src/util.cpp

```diff
@@ -427,6 +429,74 @@ bool getNewsRSS(std::vector<WebsiteNews>& WebsiteNewsList)
     return !WebsiteNewsList.empty();
 }
 
+bool prepare_bug_report_file(wxString& path)
+{
+    const std::vector<std::pair<wxString, wxString>> fixes = {
+    { "\n\n", "<br>" }, { "\n", " " }, { "  ", " " },
+    { "^" + _("Version: "), "\n<hr><small><b>Version</b>: " },
+    { _("Database version: "), L"\u2022 db " },
+    { _("Git commit: "), L"\u2022 git " },
+    { _("Git branch: "), "" },
+    { _("MMEX is using the following support products:") + L" \u2022", "<b>Libs</b>:" },
+    { "<br>" + _("Build on"), "<br><b>Build</b>:" },
+    { " " + _("with:"), "" },
+    { _("Running on:") + L" \u2022", "<b>OS</b>:" },
+    { "(.)$", "\\1</small>" }
+    };
+
+    wxRegEx re;
+    wxString diag = mmex::getProgramDescription();
+
+    for (const auto& kv : fixes)
+    {
+        if (re.Compile(kv.first, wxRE_EXTENDED)) {
+            re.Replace(&diag, kv.second);
+        }
+    }
+
+    wxString info = "> " + _("Replace this text with detailed description of your problem.") + "\n> ";
+    info << _("Please do not remove information attached below this text.") + "\n";
+
+    wxURI req = mmex::weblink::BugReport + "/new?body=" + info + diag;
+
+    const wxString texts[] = {
+        _("Use Help->Check for Updates in MMEX to get latest version, where your problem might be already fixed."),
+        wxString::Format(_("Search %s for similar problem. If so, update existing issue instead of creating a new one.")
```

Comment:
```suggestion
        wxString::Format(_("Search <a href='%s'>a list of known issues</a> for similar problem. If so, update existing issue instead of creating a new one.")
```